exec/internal/compile/scanner.go

```diff
@@ -94,7 +94,16 @@ func (s *scanner) ScanFunc(bytecode []byte, meta *BytecodeMetadata) ([]Compilati
 		case ops.I64Const, ops.GetLocal:
 			inProgress.Metrics.IntegerOps++
 			inProgress.Metrics.StackWrites++
+		case ops.SetLocal:
+			inProgress.Metrics.IntegerOps++
+			inProgress.Metrics.StackReads++
+		case ops.I64Eqz:
+			inProgress.Metrics.IntegerOps++
+			inProgress.Metrics.StackReads++
+			inProgress.Metrics.StackWrites++
 
+		case ops.I64Eq, ops.I64Ne, ops.I64LtU, ops.I64GtU, ops.I64LeU, ops.I64GeU:
```

Comment:
unless we expect additional operations to be carried for during the case, I would just merge all these cases together, possibly leaving each of these different cases sets regrouped on their own line:
```go
case ops.I64Eq, ops.I64Ne, ops.I64LtU, ops.I64GtU, ops.I64LeU, ops.I64GeU,
    ops.I64Shl, ops.I64ShrU, ops.I64ShrS,
    ops.I64DivU, ops.I64RemU, ops.I64DivS, ops.I64RemS,
    ops.I64Add, ops.I64Sub, ops.I64Mul, ops.I64And, ops.I64Or, ops.I64Xor:
```

apologies for missing this earlier :)

Reply:
Ack, will do in morning :)